docs/stencil-docs/deploying-a-theme/troubleshooting-theme-uploads.md

```diff
@@ -44,8 +44,8 @@ Custom theme uploads must meet these restrictions:
 | TR-1600 | There was a temporary problem on our system. Please try again. |
 | TR-1601 | There was a problem with processing screenshots. |
 | TR-1700, -1800, -1801, -1802, -1803 | System error, possibly temporary. Please try again. |
-| TR-3402 | You are not allowed to edit your active theme. [Please select <code>Make a Copy</code>, then edit the resulting copy of your theme.] |
-| TR-4400 | One or more values in the <code>config.json</code> file exceed the 64-character limit. (Can also indicate string lengths that exceed their limits in other JSON files.) |
+| TR-3402 | You are not allowed to edit your active theme. [Please select `Make a Copy`, then edit the resulting copy of your theme.] |
```

Comment:
```suggestion
| TR-3402 | You are not allowed to edit your active theme. [Please select **Make a Copy**, then edit the resulting copy of your theme.] |
```
Might as well change this to bold instead of monospaced while changing the HTML to MD. That is, assuming it references the menu item below
![image](https://user-images.githubusercontent.com/29524703/131899248-d58e99ac-8fad-465d-a827-9b2dc0db4fcb.png)